package.json

```diff
@@ -17,6 +17,7 @@
         "gpii-couchdb-test-harness": "1.0.0-dev.20190227T100402Z.f266632.GPII-3531",
         "gpii-json-schema": "2.0.1",
         "infusion": "3.0.0-dev.20190328T144119Z.ec44dbfab",
+        "ip-range-check": "0.0.2",
```

Comment:
Please use the "ipaddr" package directly since this is substantially better maintained and includes the required functionality already - https://github.com/whitequark/ipaddr.js - also, this spares us a possible security risk due to the dependency of ip-range-check on ipaddr itself not being version-pinned